pkg/codegen/templates/strict/strict-interface.tmpl

```diff
@@ -63,6 +63,12 @@
                 }
             {{end}}
 
+            {{if eq .NameTag "JSON" | and $fixedStatusCode $isRef -}}
```

Comment:
This is not related to the commit message. Perhaps split it out into a separate PR?